python/grass/script/__init__.py

```diff
@@ -3,6 +3,7 @@
 
 from .core import *
 from .db import *
+from .imagery import *
```

Comment:
It's not needed if you want to dot access. This only "copies" (makes available) all from the imagery module (the `__init__.py`) inside of this namespace. You also need to make sure to avoid introducing new circular imports, as Pylint 3.x complains already at multiple places (before this PR)

Reply:
I removed the line again by reverting the commit...